pkg/apis/rollouts/register.go

```diff
@@ -19,17 +19,15 @@ package rollouts
 const (
 	Group string = "argoproj.io"
 
-	RolloutKind      string = "Rollout"
-	RolloutSingular  string = "rollout"
-	RolloutPlural    string = "rollouts"
-	RolloutShortName string = "ro"
-	RolloutFullName  string = RolloutPlural + "." + Group
-
-	ExperimentKind      string = "Experiment"
-	ExperimentSingular  string = "experiment"
-	ExperimentPlural    string = "experiments"
-	ExperimentShortName string = "ex"
-	ExperimentFullName  string = ExperimentPlural + "." + Group
+	RolloutKind     string = "Rollout"
+	RolloutSingular string = "rollout"
+	RolloutPlural   string = "rollouts"
+	RolloutFullName string = RolloutPlural + "." + Group
+
+	ExperimentKind     string = "Experiment"
+	ExperimentSingular string = "experiment"
+	ExperimentPlural   string = "experiments"
+	ExperimentFullName string = ExperimentPlural + "." + Group
```

Comment:
We were no longer using the shortnames so i removed it. instead i added it as kubebuilder comments